ledger-core/Makefile.testing

```diff
@@ -15,7 +15,7 @@ test_unit: ## run all unit tests
 
 .PHONY: test_with_coverage
 test_with_coverage: ## run unit and slow tests with generation of coverage file
-	go-acc --covermode=count --output=coverage.tmp.out --ignore ledger-v2 ./... -- -tags=slowtest -count=1 $(TEST_ARGS) -json > "integration_test.log"
+	go-acc --covermode=count --output=coverage.tmp.out --ignore ledger-v2 ./... -- -v -tags="slowtest convlogtxt" -count=1 $(TEST_ARGS) 2>&1 | tee "integration_test.log"
```

Comment:
testrail can't parse txt log

Reply:
then we can prepare output for testrail with [test2json](https://golang.org/cmd/test2json/)

I would really like to see actual test logs inside checks

Reply:
ok, add please into step `publish_integration_tests`

Reply:
split make targets for CI from targets for developers

Reply:
It doesn't make sense to run `test_with_coverage` locally. So why split?

Reply:
I didn't understand @sexyhamster 's reasons to do all this. Ignore my "split" comment.

Reply:
As discussed offline, I'll need to postprocess text test output to put minimal information to json for testrail. That gives us realtime output in CI run, and testrail information. I'll read more about testrail API and will figure out how to do that. If someone have free hands or idea how to make it quick, help would be much appreciated